lib/timeline-model.js

```diff
@@ -61,9 +61,14 @@ class SandboxedModel {
     // timeline model
     this._timelineModel = new WebInspector.TimelineModel(WebInspector.TimelineUIUtils.visibleEventsFilter())
 
+    // convert string (e.g. via fs.readFileSync) to JS
+    if (typeof events === 'string') events = JSON.parse(events)
+    // WebPagetest trace files put events in object under key `traceEvents`
```

Comment:
I just learned that this isn't specific to WebPageTest. It's listed under [JSON Object format on the Trace Event Format doc](https://docs.google.com/document/d/1CvAClvFfyA5R-PhYUmn5OOQtYMH4h6I0nSsKchNAySU/preview#heading=h.q8di1j2nawlp)